features/multivariate_put_dimension.authorized.feature

```diff
@@ -183,4 +185,3 @@ Feature: Multivariate Feature Dimensions Private Endpoints
       "total_count": 2
     }
     """
```

Comment:
MIssing CR

Reply:
should be fixed now? right side shows new lines added